README.md

```diff
@@ -29,6 +29,13 @@ Extends from [passports-form-wizard](https://github.com/UKHomeOffice/passports-f
   /* step options */
 }
 ```
+
+#### Added functionality for invalidating steps
+
+Functionality exists in `hmpo-form-wizard` to provide an array of `invalidates` to a field config object. This attaches a listener to the field and on `change:{field}` it will 'invalidate' any fields passed in the `invalidates` array. This means they are removed from the sessionModel.
+
```

Comment:
Removing fields from the session when the value of another field is changed. In the following example, when the value of `a_field` is changed, `field_1` and `field_2` - and their values - will be removed from the session model. If all the fields in a step are removed in this way, the entire step is removed from the session model too.

This is useful when a user takes a fork in a journey, only to return to the fork and take the other route.
```
a_field: {
    invalidates: ['field_1', 'field_2']
}
```